src/scripts/main.js

```diff
@@ -1,3 +1,62 @@
+/* eslint-disable prefer-promise-reject-errors */
+/* eslint-disable promise/param-names */
+/* eslint-disable no-unused-vars */
 'use strict';
 
-// write your code here
+function showNotification(message, type) {
+  const notification = document.createElement('div');
+
+  notification.className = type;
+  notification.setAttribute('data-qa', 'notification');
+  notification.textContent = message;
+  document.body.appendChild(notification);
+}
+
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((_, reject) => {
+  setTimeout(() => {
+    reject('Promise was rejected!');
+  }, 3000);
+});
+
+promise1
+  .then((message) => {
+    const successDiv = document.createElement('div');
+
+    successDiv.className = 'message';
+    successDiv.textContent = message;
+    document.body.appendChild(successDiv);
+  })
+  .catch((error) => {
+    const errorDiv = document.createElement('div');
+
+    errorDiv.className = 'message error-message';
+    errorDiv.textContent = error;
+    document.body.appendChild(errorDiv);
+  });
```

Comment:
The catch block for promise1 is redundant because promise1 is only resolved and not rejected. Consider removing this catch block if you are sure promise1 will never be rejected.